internal/driver/driver.go

```diff
@@ -208,7 +208,10 @@ func (d *Driver) debouncedDiscover() {
 			d.debounceTimer = nil
 			d.debounceMu.Unlock()
 
-			d.Discover()
+			err := d.Discover()
+			if err != nil {
+				d.lc.Errorf("failed to discover device, %v", err)
```

Comment:
I think something more like: `failed to run device discovery` sounds better.

Reply:
Updated